sql/updates/latest-dev.sql

```diff
@@ -2,3 +2,109 @@ CREATE FUNCTION _timescaledb_functions.compressed_data_info(_timescaledb_interna
     RETURNS TABLE (algorithm name, has_nulls bool)
     AS '@MODULE_PATHNAME@', 'ts_update_placeholder'
     LANGUAGE C STRICT IMMUTABLE SET search_path = pg_catalog, pg_temp;
+
+DROP FUNCTION IF EXISTS _timescaledb_functions.chunk_constraint_add_table_constraint(
+    chunk_constraint_row  _timescaledb_catalog.chunk_constraint
+);
+DROP FUNCTION IF EXISTS _timescaledb_internal.chunk_constraint_add_table_constraint(
+    chunk_constraint_row  _timescaledb_catalog.chunk_constraint
+);
+
+CREATE OR REPLACE FUNCTION _timescaledb_functions.chunk_constraint_add_table_constraint(
```

Comment:
No need to add this here since we add it during the upgrade script building.

Reply:
I think I added it because there was a test failing due to its absence here.

Reply:
@fabriziomello Removed it and it does seem to pass the tests. Thanks.